CHANGES.md

```diff
@@ -18,6 +18,7 @@
 
 ##### Fixes :wrench:
 
+- Fixed the rendering issues related to order-independent translucency on iOS devices. [#10417](https://github.com/CesiumGS/cesium/pull/10417)
```

Comment:
Not sure if this is how we should phrase it or if it needs to go into breaking changes (since the workaround initially went in breaking changes)